rpc-interface/src/consensus.rs

```diff
@@ -1,11 +1,12 @@
 use async_trait::async_trait;
 
+use crate::types::{RPCResult, Transaction, ValidityStartHeight};
 use nimiq_hash::Blake2bHash;
 use nimiq_keys::Address;
 use nimiq_primitives::coin::Coin;
 use nimiq_transaction::account::htlc_contract::{AnyHash, HashAlgorithm};
 
-use crate::types::{Transaction, ValidityStartHeight};
+type Result<T, E> = core::result::Result<RPCResult<T, ()>, E>;
```

Comment:
Since you are defining this type in multiple files, it might make sense to move it to another file, make it `pub` and import it instead.
We might also think about giving it a different name but I'm terrible at names. 😅

Reply:
(probably into the `types.rs` file)

Reply:
done